$OpenBSD: patch-texk_dvipsk_dospecial_c,v 1.1 2011/11/23 12:00:45 edd Exp $

dvips, e-ptex fixes (buffer overflow/memory corruption) on tl11 branch.
http://tug.org/pipermail/tldistro/2011q4/000149.html

--- texk/dvipsk/dospecial.c.origy	Mon May 10 16:47:35 2010
+++ texk/dvipsk/dospecial.c	Sun Oct 23 20:54:17 2011
@@ -112,7 +112,7 @@ static void
 fgetboundingbox(char *f, float *llx_p, float *lly_p, float *urx_p, float *ury_p)
 {
    FILE *fp;
-   char buf[BUFSIZ];
+   char buf[BUFSIZ+1];
 
    fp = search(figpath, f, READ);
    if (fp == 0)
